assets/queries/terraform/aws/s3_bucket_with_unsecured_cors_rule/test/negative2.tf

```diff
@@ -0,0 +1,20 @@
+module "s3_bucket" {
+  source = "terraform-aws-modules/s3-bucket/aws"
+  bucket = "s3-tf-example-versioning"
+  acl    = "private"
+  version = "0.0.1"
+
+  versioning = [
+    {
+      enabled = true
+      mfa_delete = null
+    },
+  ]
+
+  cors_rule {
+    allowed_methods = ["PUT", "POST"]
+    allowed_origins = ["https://s3-website-test.hashicorp.com"]
+    expose_headers  = ["ETag"]
+    max_age_seconds = 3000
+  }
```

Comment:
I think this sample is wrong, since `cors_rule` is a list of maps, not a single map, as you can see [here](https://registry.terraform.io/modules/terraform-aws-modules/s3-bucket/aws/latest#input_cors_rule)